webui/react/src/components/Logo.tsx

```diff
@@ -12,9 +12,9 @@ import logoDeterminedOnLightVertical from
   'shared/assets/images/logo-determined-on-light-vertical.svg';
 import logoHpeOnDarkHorizontal from 'shared/assets/images/logo-hpe-on-dark-horizontal.svg';
 import logoHpeOnLightHorizontal from 'shared/assets/images/logo-hpe-on-light-horizontal.svg';
-import { DarkLight } from 'themes';
 import { BrandingType } from 'types';
 
+import { DarkLight } from '../shared/themes';
```

Comment:
**suggestion:** `shared/themes` ("absolute" instead of relative path)

Reply:
these are how webstorm puts them (automated) I'm thinking of writing a script or finding a tool that updates these for us.

Reply:
(already included as part of [this ticket ](https://determinedai.atlassian.net/browse/DET-7628?atlOrigin=eyJpIjoiMjliZDhjZmFmZGNjNDQ4OGExNTM3NDYzNjkwMGVmZjYiLCJwIjoiamlyYS1zbGFjay1pbnQifQ) we have other instances of this